app/services/mapping_migration_service.rb

```diff
@@ -0,0 +1,90 @@
+class MappingMigrationService
+  attr_reader :integration_name, :rtp_id, :template_file, :templates_dir, :upload_integrations
+
+  def initialize
+    @templates_dir = Configuration.paths_templates_plugins
+    @upload_integrations = Dradis::Plugins::with_feature(:upload)
+  end
+
+  def call
+    upload_integrations.each do |integration|
+      @integration_name = integration.plugin_name.to_s
+
+      template_files.each do |template_file|
+        @template_file = template_file
+
+        if defined?(Dradis::Pro)
+          migrate_pro
+        else
+          migrate
+        end
+        # delete the .template files after migrating them to the db
+        File.delete(template_file)
```

Comment:
Are we 100% sure about making this is irreversible action of completely deleting the templates? What about we simply rename templates and give them `.old` (e.g. `evidence.template.old`) in the likely scenario that something goes wrong?

Reply:
You're right, would be good to have the safety of this